crypto/keyring/keyring.go

```diff
@@ -107,13 +107,18 @@ type Signer interface {
 	SignByAddress(address sdk.Address, msg []byte) ([]byte, types.PubKey, error)
 }
 
-// Importer is implemented by key stores that support import of public and private keys.
+// Importer is implemented by key stores that support import of info, public and private keys.
 type Importer interface {
 	// ImportPrivKey imports ASCII armored passphrase-encrypted private keys.
 	ImportPrivKey(uid, armor, passphrase string) error
 
 	// ImportPubKey imports ASCII armored public keys.
 	ImportPubKey(uid string, armor string) error
+
+	// ImportInfo takes a keyring.Info (in practise, from an old keyring), and
+	// writes it to the current keyring. We use it to migrate Type{Multi,Ledger,Offline}
+	// keyring.Infos.
+	ImportInfo(oldInfo Info) error
```

Comment:
This is breaking too. Now I remember why we had created the notorious `InfoImporter` interface @jgimeno

Reply:
Ah, right. As a workaround, we can temporarily do `migrator.(keyring.Keyring).ImportInfo` on [this line](https://github.com/cosmos/cosmos-sdk/pull/8639/files#diff-a2bc36b1542fa363d2fd3f4c1f2789d5e82d87f9836f30f6b6cbfff9bdeefbabR116), and remove this interface change. wdyt @sahith-narahari @jgimeno

Reply:
It works for me

Reply:
do you want me to add ImportInfo as a method of keyring?

Reply:
In this PR, `ImportInfo` is already a method on keyring, so we keep it. we just remove it from the interface, as to not introduce a breaking change.

Reply:
Unless the keyring is not exported I think we cannot cast a method that is not in the interface, the only way is creating the InfoImporter back.

Reply:
> Unless the keyring is not exported I think we cannot cast a method that is not in the interface, the only way is creating the InfoImporter back.

I think @amaurym's solution (https://github.com/cosmos/cosmos-sdk/pull/8639/files#r583586459) should work. Wdyt?